src/open_clip/transformer.py

```diff
@@ -272,9 +272,24 @@ def __init__(
         self.init_parameters()
 
     def lock(self, unlocked_groups=0, freeze_bn_stats=False):
-        assert unlocked_groups == 0, 'partial locking not currently supported for this model'
         for param in self.parameters():
             param.requires_grad = False
+        
+        if unlocked_groups != 0:
+            modules = [
+                self.conv1,
```

Comment:
ross advises to put self.conv1 self.class_embedding self.positional_embedding as one